FIAT/gen_quad_data.py

```diff
@@ -0,0 +1,5823 @@
+# GENERATED, DO NOT EDIT
+# Slobodkins-Tausch quadratures
```

Comment:
Suggest linking to the code repo (with a commit hash), so that someone else knows what was run to generate this.